readme.md

```diff
@@ -198,8 +203,7 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
 * **Question:** Describe the purpose of both morgan and body-parser. Why do you think morgan is only being run when the application is run in development mode?
 
 * **Your Answer:**
-
----
+* If morgan is signed in he or she will have dev environment variable. The body-parser appears to set the environment variable to json().  Meaning it's parsing all data to json. 
```

Comment:
Morgan provides logs which are not very useful outside of development. 

without body-parser we would lose access to req.body